kotlin/openId4VP/publish-artifact.gradle

```diff
@@ -0,0 +1,94 @@
+publishing {
+    repositories {
+        maven {
+            url = System.getenv("OSSRH_URL")
+            name = "inji-openId4VP"
+            credentials {
+                username = System.getenv("OSSRH_USER")
+                password = System.getenv("OSSRH_SECRET")
+            }
+        }
+    }
+    publications {
+        aar(MavenPublication) {
+            pom {
+                withXml {
```

Comment:
Please refer secure-keystore publish-artifact.gradle to have all mandatory fields in pom xml